src/hooks/useLifeOmicConfig.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TextStyles, defaultTextStyles } from 'src';
+import * as theme from '../theme';
+
+export interface AppConfig {
+  styles: {
+    Text?: TextStyles;
+  };
+  theme: {};
+}
+
+const LifeOmicConfigContext = React.createContext<AppConfig>({
+  styles: {
+    Text: defaultTextStyles,
+  },
+  theme,
+});
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export function LifeOmicConfigProvider({ children }: Props) {
+  // TODO: override default styles with local consumer and config service
+  const styles = {
+    Text: defaultTextStyles,
+  };
+
+  const context = {
+    styles,
+    theme,
+  };
+
+  return (
+    <LifeOmicConfigContext.Provider value={context}>
+      {children}
+    </LifeOmicConfigContext.Provider>
+  );
+}
+
+export const useLifeOmicConfig = () => React.useContext(LifeOmicConfigContext);
```

Comment:
Let's stick with `useAppConfig` when we get to that hook - but this seems different.  Can this be `useBranding`?